src/graph/visitor/PropertyTrackerVisitor.cpp

```diff
@@ -279,8 +279,10 @@ void PropertyTrackerVisitor::visit(AttributeExpression *expr) {
 }
 
 void PropertyTrackerVisitor::visit(FunctionCallExpression *expr) {
+  // length function support `STRING` input too, so we can't ignore it directly
+  // TODO add type info to variable to help optimize it.
   static const std::unordered_set<std::string> ignoreFuncs = {
-      "src", "dst", "type", "typeid", "id", "rank", "length"};
+      "src", "dst", "type", "typeid", "id", "rank" /*, "length"*/};
```

Comment:
Could we check if the argument type is path？

Reply:
Could do it in later optimization. This PR just fix bug.

Reply:
I remember It was to optimze length(path) related query in ldbc. @nevermore

Reply:
yes, we can add the type of the variable to the operator later, which can be more precise when clipping attributes